jenkins/scripts/integration_test.sh

```diff
@@ -57,12 +60,21 @@ EXT_PORT_ID="$(openstack port create -f json \
   --fixed-ip subnet="$(get_subnet_name "${CI_EXT_NET}")" \
   "${TEST_EXECUTER_PORT_NAME}" | jq -r '.id')"
 
-# Create new executer vm
-openstack server create -f json \
-  --image "${IMAGE_NAME}" \
+# Get the base volume
+BASE_VOLUME_NAME_ID="$(openstack volume show -f json \
+  "${BASE_VOLUME_NAME}" | jq -r '.id')"
+
+# Create copy of base volume
+TEST_EXECUTER_VOLUME_ID="$(openstack volume create -f json \
+  --source "${BASE_VOLUME_NAME_ID}" \
+  "${TEST_EXECUTER_VOLUME_NAME}" | jq -r '.id')"
```

Comment:
you should specify a size here probably, since 50GB won't be enough

Reply:
Also since you use a volume, you probably do not need the volume part of https://github.com/metal3-io/project-infra/pull/126/files#diff-df6d7fae6b1eb79d8eff8d9f7869a586R50 and https://github.com/metal3-io/project-infra/pull/126/files#diff-df6d7fae6b1eb79d8eff8d9f7869a586R53

Reply:
> you should specify a size here probably, since 50GB won't be enough

I have specified a size of 200GB

> Also since you use a volume, you probably do not need the volume part of https://github.com/metal3-io/project-infra/pull/126/files#diff-df6d7fae6b1eb79d8eff8d9f7869a586R50 and https://github.com/metal3-io/project-infra/pull/126/files#diff-df6d7fae6b1eb79d8eff8d9f7869a586R53

Sorry, but there are no volume parts in the links you are referencing to?

Reply:
it's because you need to expand the file. Anyways, I meant this :
```
if [ "${TESTS_FOR}" == "feature_tests" ]
then
  TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
fi
TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-16GB-200GB}"
```

Reply:
Done.